test/com/facebook/buck/android/AndroidInstrumentationTestIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2015-present Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may obtain
+ * a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.facebook.buck.android;
+
+import com.facebook.buck.testutil.integration.DebuggableTemporaryFolder;
+import com.facebook.buck.testutil.integration.ProjectWorkspace;
+import com.facebook.buck.testutil.integration.TestDataHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.io.IOException;
+
+public class AndroidInstrumentationTestIntegrationTest {
+
+  @Rule
+  public DebuggableTemporaryFolder tmpFolder = new DebuggableTemporaryFolder();
+
+  @Test
+  public void testAndroidInstrumentationTest() throws IOException {
+    tmpFolder.doNotDeleteOnExit();
+    ProjectWorkspace workspace = TestDataHelper.createProjectWorkspaceForScenario(
+        new AndroidBinaryIntegrationTest(),
+        "android_instrumentation_test_integration_test",
+        tmpFolder);
+    workspace.setUp();
+    workspace.runBuckCommand("test").assertSuccess();
```

Comment:
We need to only run when the android sdk is present.  I'll add the appropriate assume statement here.